src/Compilers/Core/Portable/Emit/EditAndContinue/DeltaMetadataWriter.cs

```diff
@@ -582,18 +599,18 @@ void addDeletedClosureMethods(IMethodSymbolInternal oldMethod, ImmutableArray<En
 
                             if (deletedClosureMethod.MetadataToken != 0)
                             {
-                                newMethodDefs.Add(new DeletedPEMethodDefinition(deletedClosureMethod, il));
+                                newMemberDefs.Add(new DeletedPEMethodDefinition(deletedClosureMethod, il));
                             }
                             else
                             {
                                 var deletedClosureMethodDef = (IMethodDefinition)deletedClosureMethod.GetCciAdapter();
                                 var deletedClosureMethodHandle = changes.DefinitionMap.GetPreviousMethodHandle(deletedClosureMethod);
-                                newMethodDefs.Add(new DeletedSourceMethodDefinition(deletedClosureMethodDef, deletedClosureMethodHandle, il, typesUsedByDeletedMembers));
+                                newMemberDefs.Add(new DeletedSourceMethodDefinition(deletedClosureMethodDef, deletedClosureMethodHandle, il, typesUsedByDeletedMembers, deletedAttribute: null));
```

Comment:
We don't need to mark deleted closure methods with MetadataUpdateDeletedAttribute?